### DISCUSSION TITLE
When will Access Control APIs be added?

### DISCUSSION THREAD

**Discussion by ideal** [unitycatalog/unitycatalog#426] [Ideas]
**Is your feature request related to a problem? Please describe.**

No.

**Describe the solution you would like**

As https://www.databricks.com/blog/open-sourcing-unity-catalog says, Access Control APIs is in the roadmap, so when will it be added?

**Additional context**

None.

**Comment by zpappa** [comment]
We should have support for access control in V0.2, end of August. I'll post some details in #229 soon

**Comment by ideal** [comment]
> We should have support for access control in V0.2, end of August. I'll post some details in #229 soon

Will it have authorization API on objects for role or user like this: https://docs.databricks.com/en/data-governance/unity-catalog/index.html#granting-and-revoking-access-to-database-objects-and-other-securable-objects-in-unity-catalog ?

**Reply by creechy** [reply]
Access control/Authorization APIs did show up in V0.2. The API should be a subset of permissions available in Databricks UC. There currently no DML to manipulate permissions, only the `uc` CLI and REST API.
